<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(166, 214, 214);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CulcRes">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TypeOp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(166, 214, 214);
</string>
    </property>
    <item>
     <property name="text">
      <string>addition</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>subtraction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>division</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>multiplication</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="Sign1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 156);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="Sign2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 156);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	qproperty-aligment:'AlignVCenter | AlignRight';
}
background-color: rgb(57, 58, 61);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_arccos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>arccos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_arcsin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>arcsin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_arctg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>arctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_arcctg">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>arcctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ctg">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>ctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>tg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(165, 143, 170);
	
	
	color: rgb(237, 237, 208);
	border: 1px solid rgb(150, 135, 155);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0  rgb(155, 153, 160), stop: 1  rgb(89, 123, 182));
}
QPushButton:hover{
	background-color: rgb(150, 135, 165);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deg1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 237, 208);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="min1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 237, 208);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sec1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 237, 208);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="deg2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 237, 208);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="min2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 237, 208);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sec2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 237, 208);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 144);</string>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 144);</string>
    </property>
    <property name="text">
     <string>'</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 144);</string>
    </property>
    <property name="text">
     <string>''</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 144);</string>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 144);</string>
    </property>
    <property name="text">
     <string>'</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(124, 107, 144);</string>
    </property>
    <property name="text">
     <string>''</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
